src/templates/utils.js

```diff
@@ -20,9 +20,10 @@ export async function loadLanguageAsync (i18n, locale) {
         try {
           const module = await import(/* webpackChunkName: "lang-[request]" */ '~/<%= options.langDir %>' + file)
           const messages = module.default ? module.default : module
-          i18n.setLocaleMessage(locale, typeof messages === 'function' ? await Promise.resolve(messages()) : messages)
+          let result = typeof messages === 'function' ? await Promise.resolve(messages()) : messages
```

Comment:
`const result` is better I guess

Reply:
Thanks for the feedback. I changed that.